packages/dds/matrix/src/handlecache.ts

```diff
@@ -72,11 +72,10 @@ export class HandleCache implements IVectorConsumer<Handle> {
 	/**
 	 * Used by {@link HandleCache.cacheMiss} to retrieve handles for a range of positions.
 	 */
-	private getHandles(start: number, end: number): Handle[] {
+	private getHandles(start: number, end: number, handles: Handle[]): Handle[] {
```

Comment:
Since we're mutating the input list, do we need to return it also?

Reply:
Curious, why? I think it makes sense if the new list was created (like in case of Array.map()), but here we are mutating inplace

Reply:
https://github.com/microsoft/FluidFramework/pull/24259